test/pageobjects/cartPage.js

```diff
@@ -0,0 +1,13 @@
+import Page from "./page.js";
+
+class CartPage extends Page{
+    get tblCart() { return $('.shop_table.shop_table_responsive.cart'); }
+    get buttonProceedToCheckout() { return $('.checkout-button.button.alt.wc-forward.wp-element-button'); }
+    async scrollToTblCart() {
```

Comment:
Let's add some space after "getters" section, so that we can clearly see what are getters (locators) and what are setters (methods). So just add one new line here.